thirdeye/thirdeye-pinot/src/test/java/com/linkedin/thirdeye/tools/CleanupAndRegenerateAnomaliesTool.java

```diff
@@ -1,287 +0,0 @@
-package com.linkedin.thirdeye.tools;
```

Comment:
Don't delete this utility class. We still need this tool to do adhoc cleaning and regeneration of anomalies. Since you have copied the code to OnboardingResource, you could invoke those methods from this class. Otherwise, we have to start an anomaly application in order to clean or regenerate anomalies, which is really inconvenient.